src/routes/NotFound/NotFound.scss

```diff
@@ -1,26 +1,185 @@
-@import "constants/style.scss";
+@import "src/constants/style";
 
-$logo-height: 60px;
-$text-size--medium: 24px;
+.not-found__root {
+  height: 100%;
+  width: 100%;
+  padding: 20px 0 20px 20px;
+  overflow-x: hidden;
+  background: linear-gradient(320deg, rgb(107, 156, 255, 30%), rgb(255, 255, 255, 30%));
+}
+
+.not-found__background {
+  position: relative;
+}
+
+.not-found__background-form {
+  position: absolute;
+}
+
+.not-found__background-details {
+  position: fixed;
+  width: 100%;
+  height: 100%;
+}
+
+.not-found__scrumlr-logo--web,
+.not-found__scrumlr-logo--web {
+  width: 160px;
+  display: none;
+}
+
+.not-found__header {
+  position: absolute;
+}
 
-.not-found {
+.not-found__main {
   display: flex;
-  flex-direction: column;
+  flex-direction: column-reverse; // in mobile, stan is above the content
+  gap: 20px;
+
+  height: 100%;
+
   justify-content: center;
   align-items: center;
-  height: 100%;
-  width: 100%;
 }
 
-.not-found__logo {
-  height: $logo-height;
+// content (left)
+.not-found__content {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start; // prevent button from taking the whole space
+
+  padding: 0 $padding--default;
+
+  z-index: $base-z-index; // layer content over background, using any z-index works
 }
 
-.not-found__text {
-  color: $color-black;
-  font-size: $text-size--medium;
-  letter-spacing: $letter-spacing--medium;
+.not-found__title {
+  font-weight: 700;
+  font-size: 80px;
 }
-[theme="dark"] .not-found__text {
+
+.not-found__description {
+  margin-top: $margin--default;
+
+  font-weight: 400;
+  font-size: 38px;
+  line-height: 50px;
+}
+
+.not-found__return-button {
```

Comment:
I'd personally center the button on the screen for mobile devices. What do you think?